libs/iostability/iostability_lib.py

```diff
@@ -132,7 +132,11 @@ def run(self):
             subject = f"Test {self.test_id} in progress on {self.health_obj.hostname}"
             body = f"hctl Status: {status} \n"
             self.mail_obj.send_mail(subject=subject, body=body)
-            time.sleep(self.interval * 60 * 60)
+            current_time = time.time()
+            while time.time() < current_time + self.interval * 60 * 60:
+                if self.event_pass.is_set() or self.event_fail.is_set():
+                    break
+                time.sleep(60)
```

Comment:
can we take from yaml?

Reply:
This is used for polling if event if set. We can keep it constant.